ui/static/ui/js/pagination.jsx

```diff
@@ -0,0 +1,113 @@
+define('pagination', ['jquery', 'lodash', 'react'], function ($, _, React) {
+  'use strict';
+
+  var Pagination = React.createClass({
+    mixins: [React.addons.LinkedStateMixin],
+    getInitialState: function() {
+      return {
+        isEditing: false,
+        textPageNum: this.props.pageNum
+      };
+    },
+    render: function() {
+      if (this.props.numPages <= 1) {
```

Comment:
I was suggesting to check that `this.props.numPages` is an integer and not float
but I checked that it does not create weird problem (it gives a 404), so feel free to not consider this comment